go/consensus/api/api.go

```diff
@@ -375,6 +375,9 @@ type Status struct { // nolint: maligned
 
 	// P2P is the P2P status of the node.
 	P2P *P2PStatus `json:"p2p,omitempty"`
+
+	// CheckpointHeights is the list of node's local checkpoint heights.
+	CheckpointHeights []uint64 `json:"checkpoint_heights,omitempty"`
```

Comment:
We could put this under `CheckpointStatus` or `SyncStatus`, as we might add additional fields latter. For example, we could display a list of local heights and a list of P2P heights.